src/main/java/se/kth/sda/skeleton/user/User.java

```diff
@@ -94,4 +102,20 @@ public void setCreatedPosts(List<Post> createdPosts) {
     public void setLikedPosts(List<PostLike> likedPosts) {
         this.likedPosts = likedPosts;
     }
+
+    public List<PostLike> getLikedComments() {
+        return likedComments;
+    }
+
+    public void setLikedComments(List<PostLike> likedComments) {
+        this.likedComments = likedComments;
+    }
```

Comment:
Please remove this we will implement later

Reply:
it's been removed